setup_env.sh

```diff
@@ -13,35 +13,41 @@ NATIVE_LIB="ON"
 COMPILER_SETTING="-DCMAKE_CXX_COMPILER=${CXX:-"g++"} -DCMAKE_C_COMPILER=${CC:-"gcc"}"
 # Avoid having the vendors compiled in the same directory
 EXTENSION_CC=${CC:-"gcc"}
+# strip version number from compiler
+EXTENSION_CC=${EXTENSION_CC%-*}
 EXTENSION_OS=${OS_IDENTIFIER:-"linux"}
 VENDOR_EXTENSION="-DVENDOR_EXTENSION=_${EXTENSION_CC,,}_${EXTENSION_OS,,}"
-COMMON_OPT="${COMPILER_SETTING} ${VENDOR_EXTENSION} -DACCURACY_TEST=ON -DCMAKE_INSTALL_PREFIX=${DDPROF_INSTALL_PREFIX} -DBUILD_BENCHMARKS=${DDPROF_BUILD_BENCH} -DBUILD_NATIVE_LIB=${NATIVE_LIB}"
+COMMON_OPT="${COMPILER_SETTING} ${VENDOR_EXTENSION} -GNinja -DACCURACY_TEST=ON -DCMAKE_INSTALL_PREFIX=${DDPROF_INSTALL_PREFIX} -DBUILD_BENCHMARKS=${DDPROF_BUILD_BENCH} -DBUILD_NATIVE_LIB=${NATIVE_LIB}"
 
 RelCMake() {
+    # shellcheck disable=SC2086
```

Comment:
This is for things to work with zsh ?

Reply:
This suppresses a warning from [shellcheck](https://github.com/koalaman/shellcheck) when word splitting is actually wanted.
shellcheck is a great linter for shell scripts !